src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -50,7 +50,7 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
     name: "",
   });
 
-  const eventSpaceId = event?.spaceId || venue?.id;
+  const eventSpaceId = event?.spaceId || (venue?.id as SpaceId);
```

Comment:
Please remember the `undefined` otherwise TS will override the `?.` in checks.

```suggestion
  const eventSpaceId = event?.spaceId || (venue?.id as SpaceId|undefined);
```

Reply:
added: https://github.com/sparkletown/sparkle/pull/2969/commits/54d690c68fa6b074181f6d18a88fb443a9797d0d